src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double side1;
+    private final double side2;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double side1, double side2, double height) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (side1 + side2) / 2 * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid,"
```

Comment:
Instead of directly calling `System.out.println` in the `draw` method, consider abstracting the display logic to a separate method or class to follow the single responsibility principle and to allow for easier changes to the display logic in the future.